src/app/pages/component-sidenav/component-sidenav.html

```diff
@@ -11,7 +11,7 @@
   <div class="docs-component-sidenav-content">
     <component-page-header (toggleSidenav)="toggleSidenav(sidenav)"></component-page-header>
     <div class="docs-component-sidenav-inner-content">
-      <main class="docs-component-sidenav-body-content">
+      <main class="docs-component-sidenav-body-content" id="main-content" role="main" aria-label="Side navigation bar">
```

Comment:
Is this the correct label? It looks like this is the main body contents, not the navigation bar.

Reply:
A `main` element also doesn't need to have an additional `role="main"` defined.